OptimizelySDK.Tests/OptimizelyTest.cs

```diff
@@ -1245,6 +1247,25 @@ public void TestGetFeatureVariableStringFRCulture()
             Assert.AreEqual(stringValueFR, "cta_1");
         }
 
+        [Test]
+        public void TestGetFeatureVariableJSONFRCulture()
+        {
+            SetCulture("en-US");
+            var fallbackConfigManager = new FallbackProjectConfigManager(Config);
+
+            var optimizely = new Optimizely(fallbackConfigManager);
+
+            var optimizelyJsonValue = optimizely.GetFeatureVariableJSON("string_single_variable_feature", "json_var", "testUser1");
+
+            Assert.AreEqual(optimizelyJsonValue.GetValue<long>("int_var"), 1);
```

Comment:
I will suggest, use TestData.CompareResult. 
```var expectedPayload = new Dictionary<int, object>() {
  {"int_var", 1},
  {"boolean_key", false}
};
var actualValue = optimizelyJsonValue.ToDictionary();
// Compare them.
```
The reason I am telling you, it will give false impression if GetValue is not working as expected.